ambry-tools/src/main/java/com.github.ambry/store/DumpDataHelper.java

```diff
@@ -21,21 +21,23 @@
 import com.github.ambry.messageformat.MessageFormatException;
 import com.github.ambry.messageformat.MessageFormatRecord;
 import com.github.ambry.utils.SystemTime;
+import com.github.ambry.utils.Time;
 import com.github.ambry.utils.Utils;
 import java.io.DataInputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.RandomAccessFile;
 import java.nio.ByteBuffer;
 import java.nio.channels.Channels;
-import java.util.concurrent.TimeUnit;
 
 
 /**
  * Helper class to assist in dumping a single blob record from a log file
  */
 class DumpDataHelper {
 
+  private static final long TIME_MS = SystemTime.getInstance().milliseconds();
```

Comment:
this was a hack I put in. Won't work if this class is loaded just once and tools are run many times. It works when we run tools one time but not if you want to have a service that runs them constantly. You will need to find a way to pass this in.